docs/nodes/qgb-binary.mdx

```diff
@@ -6,6 +6,7 @@ description: This guide shows you how to install the QGB binary.
 # Install the QGB binary
 
 import constants from "../../versions/constants.js";
+import InlineText from "../../src/components/InlineText.js";
```

Comment:
this is adding a missing import to show the variable in docs, which currently doesn't show the go version
linked issue is #951

Reply:
resolved in leap's next release.